kahuna/public/js/usage-rights/usage-rights-editor.js

```diff
@@ -56,6 +57,16 @@ usageRightsEditor.controller(
           }
       });
 
+      // @return Stream.<Array.<Category>>
+      const displayCategories$ = usageRights$.combineLatest(categories$, (urs, cats) => {
+          const uniqueCats = getUniqueCats(urs);
+          if (uniqueCats.length === 1) {
+              return cats;
+          } else {
+              return [multiCat].concat(cats);
```

Comment:
this returns a double multiCat, since the `categories$` stream also prepends it if there's more than one category

<img width="249" alt="image" src="https://github.com/user-attachments/assets/07578243-773a-4571-ab00-23466cce62e1">

Reply:
Thanks - will fix now

Reply:
Have fixed by removing concat from the original stream which is more in keeping with the original coding.